docs/modules/ROOT/pages/reference/extensions/kamelet.adoc

```diff
@@ -37,3 +37,44 @@ Or add the coordinates to your existing project:
 ----
 
 Check the xref:user-guide/index.adoc[User guide] for more information about writing Camel Quarkus applications.
+
+== Usage
+
+=== Pre-load Kamelets at build-time
+
+This extension allows to pre-load a set of Kamelets at build time using the `quarkus.camel.kamelet.identifiers` property.
+
+=== Using the Kamelet Catalog
+
+A set of pre-made Kamelets can be found on the https://camel.apache.org/camel-kamelets/latest[Kamelet Catalog].
+To use the Kamelet from the catalog you need to copy their yaml definition (that you can find https://github.com/apache/camel-kamelets/[in the camel-kamelet repo]) on your project in the classpath. Alternatively you can add the `camel-quarkus-kamelets-catalog` artifact to your `pom.xml`:
+
+[source,xml]
+----
+<dependency>
+    <groupId>org.apache.camel.kamelets</groupId>
+    <artifactId>camel-kamelets-catalog</artifactId>
```

Comment:
The artifact to include is `camel-quarkus-kamelets-catalog` or `camel-kamelets-catalog` ?

Reply:
camel-kamelets-catalog (https://github.com/apache/camel-kamelets/tree/main/library/camel-kamelets-catalog)

Reply:
is `camel-kamelets-catalog`, the problem is it will be released in the near future.